python/bindings/openravepy_global.cpp

```diff
@@ -1419,6 +1419,16 @@ void init_openravepy_global()
 // #endif
     ;
 
+#ifdef USE_PYBIND11_PYTHON_BINDINGS
+    m.def("GetMilliTime", utils::GetMilliTime, "get millisecond time");
```

Comment:
@tgn3000 you should really expose `GetMilliTime64` instead

Reply:
```
    m.def("GetMilliTime", utils::GetMilliTime64, "get millisecond time");
```